src/collection-browser.ts

```diff
@@ -1057,6 +1109,8 @@ export class CollectionBrowser
         --collectionBrowserCellMinHeight,
         5rem
       );
+      /* 30px, but compensating for a -5px margin */
+      --infiniteScrollerRowGap: 35px;
```

Comment:
Q: Where is that -5px margin coming from?

Reply:
Added comment